charts/tidb-backup/values.yaml

```diff
@@ -13,12 +13,19 @@ mode: backup # backup | restore | scheduled-restore
 # name is the backup dir name and pvc name for ad-hoc backup and restore
 name: fullbackup-{{ date "200601021504" .Release.Time }}
 
+# The default jobName is clusterName-name
+# jobName:
+
+# The TiDB host to connect to
+# By default this will match a normal tidb-operator deploy
+# host:
+
 # scheduledBackupName is the name of a scheduled backup directory,
 # used to restore the tidb cluster from scheduled backup.
 # scheduledBackupName: scheduled-backup-20190822-041004
 
 image:
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
Why change this to `Always`?

Reply:
For infrequent long-running operations I think Always might be better. If the image is already there, which is unlikely except in the case of retry, then IfNotPresent will save doing some requests to the registry (presumably this would take a few seconds). But the problem is that a different image may be used depending on which node the pod is scheduled to. So the behavior of Always is more consistent. If we use an image tag of a specific non-changing version, then switching back to IfNotPresent is preferably to avoid contacting the registry in some cases.

Reply:
Yes, the image content will not be updated with the specific tag. Changing to `Always` will fail if there is not internet connection which is most of the case for on-prem deployment. Users may simply load the image on the backup node.

Reply:
Okay, I didn't know about that use case